kms/api-client/asymmetric.py

```diff
@@ -102,11 +103,11 @@ def signAsymmetric(message, client, key_path):
 def verifySignatureRSA(signature, message, client, key_path):
     """
     Verify the validity of an 'RSA_SIGN_PSS_2048_SHA256' signature for the
-    specified plaintext message
+    specified message
     """
     public_key = getAsymmetricPublicKey(client, key_path)
 
-    digest_bytes = hashlib.sha256(message.encode('ascii')).digest()
+    digest_bytes = hashlib.sha256(message).digest()
     sig_bytes = base64.b64decode(signature)
```

Comment:
The signature passed to verifySignature{RSA,EC} should be raw binary. signAsymmetric should b64decode the signature received in the response.

Reply:
fixed